api/helper/responses.helper.js

```diff
@@ -0,0 +1,45 @@
+module.exports.responseHelper = {
+  responseMapper: (rawData) => {
+    try {
+      let isArray = true;
+      if (!Array.isArray(rawData)) {
+        rawData = [rawData];
+        isArray = false;
+      }
+      const fixedData = rawData.map((data) => {
+        const keys = Object.keys(data);
+        const formattedObj = keys.reduce((structuredData, key) => {
+          const words = key.split("_");
+          const camelCasedWords = words.map((word, index) => {
+            if (index > 0) {
+              const chars = word.split("");
+              chars[0] = chars[0].toUpperCase();
+              return chars.reduce((newKey, char) => {
```

Comment:
Too much nested lambdas going on here - reduce inside a map inside a reduce inside a map, not even Brendan Eich could understand this.